src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -72,17 +69,25 @@ export class UiSettingsClient implements IUiSettingsClient {
 
   async get<T = any>(key: string): Promise<T> {
     const all = await this.getAll();
-    return all[key];
+    return all[key] as T;
   }
 
   async getAll<T = any>() {
-    const raw = await this.getRaw();
+    const result: Record<string, any> = {};
```

Comment:
Can we use `unknown` instead of `any` here?
```suggestion
    const result: Record<string, unknown> = {};
```